tools/delete_timepoint.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/**
+ * This script deletes the specified candidate timepoint.
+ *
+ * Delete all timepoint rows for a given candidate
+ * echo "Usage: php delete_timepoint.php delete_timepoint DCCID PSCID SessionID [confirm]";
+ * echo "Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482";
+ * echo "Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482 confirm";
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Various <example@example.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once __DIR__ . "/generic_includes.php";
+
+/**
+ * This script deletes the specified candidate timepoint.
+ *
+ * Delete all timepoint rows for a given candidate
+ * echo "Usage: php delete_timepoint.php delete_timepoint DCCID PSCID SessionID [confirm] [tosql]";
+ * echo "Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482";
+ * echo "Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482 confirm";
+ * echo "Example: php delete_timepoint.php delete_timepoint 965327 dcc0007 482 tosql";
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Various <example@example.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+
+// Possible script actions
+$actions = array('delete_timepoint');
+
+//define the command line parameters
+if (count($argv) < 4 || $argv[1] == 'help' || !in_array($argv[1], $actions)) {
+    showHelp();
+}
```

Comment:
@ridz1208 I think he told me that warning is not specific to your script, so it's not a problem